design/operator_design.md

```diff
@@ -0,0 +1,129 @@
+    podManagementPolicy: Parallel       # Optional. default value "Parallel", out of valid values "Parallel" and "OrderedReady". If "OrderedReady" value is specified, cluster will take more time coming up.
+    storage:
+      count: 2
+      size: 10Gi
+      storageClass: standard            # Optional. default value "standard". Provide storage class to use.
+    resources:
+      requests:
+        cpu: 2
+        memory: 1Gi
+      limits:
+        cpu: 2
+        memory: 1Gi
+    gflags:
+      - key: default_memory_limit_to_ram_ratio
+        value: 0.85
+  tserver:
+    # Replica count for TServer.
+    replicas: 3
+    # Mentioning network ports is optional. If some or all ports are not specified, then they will be defaulted to below-mentioned values, except for tserver-ui.
+    # For tserver-ui a cluster ip service will be created if the yb-tserver-ui port is explicitly mentioned. If it is not specified, only StatefulSet & headless service will be created for TServer. TServer ClusterIP service creation will be skipped. Whereas for Master, all 3 kubernetes objects will always be created.
+    tserverUIPort: 9000                 # Optional. ClusterIP service will not be created, if omitted.
+    tserverRPCPort: 9100                # Optional. default value "9100".
+    ycqlPort: 9042                      # Optional. default value "9042".
+    yedisPort: 6379                     # Optional. default value "6379".
+    ysqlPort: 5433                      # Optional. default value "5433".
+    enableLoadBalancer: false           # Optional. default value "false". Change it to true to be able to access YugabyteDB TServer UI over the internet. Value will be ignored if "tserverUIPort" is omitted.
+    # Volume claim template for TServer
+    podManagementPolicy: Parallel
+    storage:
+      count: 2
+      size: 10Gi
+      storageClass: standard            # Optional. Provide storage class to use. Field will be defaulted to "standard", if omitted.
+    resources:
+      requests:
+        cpu: 2
+        memory: 1Gi
+      limits:
+        cpu: 2
+        memory: 1Gi
+    gflags:
+      - key: default_memory_limit_to_ram_ratio
+        value: 0.85
+```
+
+# Cluster Settings
+
+### Image
+Mention YugabyteDB docker image attributes such as `repository`, `tag` and `pullPolicy` under `image`.
+
+### TLS
+Enable TLS encryption for YugabyteDB (Enterprise Edition only). Before setting this up, you need to have the pull secret installed from Yugabyte private Enterprise Edition registry. You can use the TLS encryption with 3 GFlags, explained later. If you have set `enabled` to true, specify your `cert` & `key` under `rootCA`.
+
+[comment]: <> (Not clear on its implementation yet. Q: How will the user know master address before creating a cluster? If current is the first cluster out of a multi-AZ deployment, then the user won't have any master addresses to specify.)
```

Comment:
They should use the service name if inside a k8s pod, or use the load balancer for outside.

Reply:
Removed multi-AZ feature from version 1, as discussed.